loafwallet/TransferAmountViewModel.swift

```diff
@@ -89,16 +89,17 @@ class TransferAmountViewModel: ObservableObject {
 			return
 		}
 
-		PartnerAPI
-			.shared
-			.withdrawToWallet(userID: userID,
-			                  token: token,
-			                  withdrawal:
-			                  ["amount": amount,
-			                   "wallet_address": address]) { _ in
-				// DEV: Mothballed
-				// Need information from Ternio to move forward
-			}
+		// DEV: Mothballed
+		// Need information from Ternio to move forward
+		//		PartnerAPI
+		//			.shared
+		//			.withdrawToWallet(userID: userID,
+		//			                  token: token,
+		//			                  withdrawal:
+		//			                  ["amount": amount,
+		//			                   "wallet_address": address]) { _ in
+		//
+		//			}
 	}
 
 	/// Transfer Litecoin from **Litewallet to Litecoin Card**
```

Comment:
As TransferAmountView was deleted, seems like we don't need this viewmodel anymore?